web_development_101/the_basics/introduction_to_web_development.md

```diff
@@ -53,43 +52,43 @@ These are some of the basic tools you will use regularly. You may not know what
 
 ### Motivation
 
-Learning to code is incredibly rewarding but it can also be difficult and frustrating.
+Learning to code is incredibly rewarding but can also be difficult and frustrating.
 The strongest assets you can have as a student are a desire to build, a problem-solving mind, and persistence in the face of setbacks.
 
 The web development industry has a long history of successful developers with varying backgrounds, so people tend to care more about what you've actually **built** than how you got there.
 
-So you need to build. Build and build and build and keep on building as much as you can. Build projects that try out new technologies and build things that you need.
-Once you start to get comfortable, start hacking on open source projects [(like The Odin project itself)](/contributing). The more you practice, the more you will learn about what you can do and the closer you will get to being hireable.
+So, you need to build. Build and build and build and keep on building as much as you can. Build projects to try out new technologies, and build things that you need.
```

Comment:
```suggestion
```

Reply:
Let's remove this paragrph

Reply:
Happy to do so! I wasn't a huge fan of that one. :)